### PR TITLE
One spotter for all ilks + par value. Closes #9

### LINKED ISSUE #9: Add par value to spotter making it the unique contract for every ilk

### PR DESCRIPTION
**Pull request by gbalabasquer**

### CODE REVIEW

**Comment by gbalabasquer** [comment]
I've defined `par` as dai per usd for convenience in the math but it could be changed.

**Comment by rainbreak** [comment]
Can we go with usd per dai please? For consistency with previous usage. Just use ds-math if it makes things easier.

**Comment by gbalabasquer** [comment]
@rainbreak done!

File: src/poke.sol [hunk 1]
```diff
@@ -8,44 +10,45 @@ contract PipLike {
...
 }
 
-contract Spotter {
+contract Spotter is DSMath {
     PitLike public pit;
-    bytes32 public ilk;
-    PipLike public pip;
-    uint256 public mat;
+    mapping (bytes32 => Ilk) public ilks;
+    uint256 public par = RAY; // usd per dai
```
Comment (rainbreak) [inline src/poke.sol]: It's actually "ref per dai", with the price feeds provided in "ref per gem". It just happens that we're choosing ref = usd at the outset.

Comment (gbalabasquer) [reply src/poke.sol]: You are right. Fixed

**Comment by gbalabasquer** [comment]
@rainbreak any other thing to change or are we ok to merge?

**Pull request merged**